example/graph-assoc-types.cpp

```diff
@@ -10,23 +10,27 @@
 
 using namespace boost;
 
-template < typename Graph > void
-generic_foo(Graph & g)
-{
+template < typename Graph > 
+void
+generic_foo(
+  Graph & g
+){
```

Comment:
I know I said I'd accept any sensible reformating, but please not mixed up with substantive changes, and IMO this is a horrible format :(